core/services/offchainreporting/oracle.go

```diff
@@ -136,6 +136,10 @@ func (d jobSpawnerDelegate) ServicesForSpec(jobSpec job.SpecDB) (services []job.
 		DatabaseTimeout:                        d.config.OCRDatabaseTimeout(),
 		DataSourceTimeout:                      d.config.OCRObservationTimeout(time.Duration(concreteSpec.ObservationTimeout)),
 	}
+	if d.config.Dev() {
+		// Skips config validation so we can use any config parameters we want.
```

Comment:
Specifically so I can set contractConfigTrackerPollInterval = "1s" to speed the test up.

Reply:
Worth leaving a comment to this effect?